cluster-operator/src/main/java/io/strimzi/operator/cluster/operator/assembly/KafkaAssemblyOperator.java

```diff
@@ -183,16 +184,17 @@ public KafkaAssemblyOperator(Vertx vertx, PlatformFeaturesAvailability pfa,
                 .compose(state -> state.zkHeadlessService())
                 .compose(state -> state.zkAncillaryCm())
                 .compose(state -> state.zkNodesSecret())
-                .compose(state -> state.zkNetPolicy())
                 .compose(state -> state.zkPodDisruptionBudget())
                 .compose(state -> state.zkStatefulSet())
                 .compose(state -> state.zkScaleUp())
                 .compose(state -> state.zkRollingUpdate(this::dateSupplier))
                 .compose(state -> state.zkServiceEndpointReadiness())
                 .compose(state -> state.zkHeadlessServiceEndpointReadiness())
                 .compose(state -> state.zkPersistentClaimDeletion())
+
```

Comment:
Remove this space?

Reply:
Actually, I did there the blank line intentionally. It follows on the other blank lines which are already there and separate the different components - between the Kafka steps, TO steps and EO steps. So I just followed it and added steps between Kafka and Zoo.

Reply:
yeah fine with it

Reply:
Thanks